refactor/actions.py

```diff
@@ -195,11 +195,59 @@ def _stack_effect(self) -> tuple[ast.AST, int]:
         return (self.node, 1)
 
 
+@_hint("deprecated_alias", "NewStatementAction")
```

Comment:
NIT: This was only for the old actions, we don't need a deprecated alias for a new action.